librz/include/rz_cons.h

```diff
@@ -1200,6 +1217,9 @@ RZ_API int rz_line_set_hist_callback(RzLine *line, RzLineHistoryUpCb cb_up, RzLi
 RZ_API int rz_line_hist_cmd_up(RzLine *line);
 RZ_API int rz_line_hist_cmd_down(RzLine *line);
 
+RZ_API void rz_line_undo_free(void);
```

Comment:
is this really required? I think it can be moved in `line.c` and just be static.
There is no `rz_line_undo_new` so I don't expect a user of RzCons to call `rz_line_undo_free` either.